cli/server/mptdump.go

```diff
@@ -0,0 +1,98 @@
+package server
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/nspcc-dev/neo-go/cli/options"
+	"github.com/nspcc-dev/neo-go/pkg/core/mpt"
+	"github.com/nspcc-dev/neo-go/pkg/core/storage"
+	"github.com/urfave/cli/v2"
+	"go.uber.org/zap"
+)
+
+// KVPair represents a key-value pair.
+type KVPair struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+// TraverseMPT collects key-value pairs from the TrieStore and returns them.
+func TraverseMPT(store *mpt.TrieStore) ([]KVPair, error) {
+	var kvPairs []KVPair
+	prefix := []byte{byte(storage.STStorage)}
+	rng := storage.SeekRange{Prefix: prefix}
+
+	store.Seek(rng, func(k, v []byte) bool {
+		kvPairs = append(kvPairs, KVPair{
```

Comment:
Please don't try to store it all in memory. It can be huge. Write to some output immediately.